components/PDFViewer.jsx

```diff
@@ -15,8 +15,8 @@ const PDFViewer = ({ pdf }) => {
   };
 
   useEffect(() => {
-    if (pdf) {
-      if (pdf.endsWith(".pdf")) {
+    {
+      if (pdf && (pdf.endsWith(".pdf") || pdf.search("drive.google.com"))) {
```

Comment:
you dont need any of these checks at the moment, you can just leave it at the assignment